WordPress/src/main/java/org/wordpress/android/ui/pages/PagesFragment.kt

```diff
@@ -264,6 +278,26 @@ class PagesFragment : Fragment() {
             savedInstanceState.getSerializable(WordPress.SITE) as SiteModel
         }
 
+        viewModel.authorUIState.observe(activity, Observer { state ->
+            state?.let {
+                val authorSelectionVisibility = if (state.isAuthorFilterVisible) View.VISIBLE else View.GONE
+                pages_author_selection.visibility = authorSelectionVisibility
+                pages_tab_layout_fading_edge.visibility = authorSelectionVisibility
+
+                val tabLayoutPaddingStart =
```

Comment:
We might want to consider moving this logic to the VM and utilizing UIDimen. Wdyt?

Reply:
That's a really good idea, but for time constraint, going to leave it where it is for now. Adding to my  back pocket task list.